### PR TITLE
spark setup for MacOs

### PR DESCRIPTION
**Pull request by IremErturk**

### CODE REVIEW

File: week_5_batch_processing/setup/macos.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+
+Here we'll show you how to install Spark 3.2.1 for MacOs.
+We tested it on MacOs Monterey 12.0.1, but it should work
+for other MacOs versions as well
+
+### Installing Java
+
+Ensure Brew and Java installed in your system.
+
+`xcode-select –install`
```
[the remark this answers is not in the document]
Comment (alexeygrigorev) [reply week_5_batch_processing/setup/macos.md:12]: And in the other code blocks below

**Comment by alexeygrigorev** [comment]
Thanks a lot!

File: week_5_batch_processing/setup/macos.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+1. Install Scala
+
+`brew install scala@2.11`
+
+2. Install Apache Spark
+
+`brew install apache-spark`
+
+3. Add Environment Variables: 
+Add the following environment variables to your `.bash_profile` or `.zshrc`
```
Comment (alexeygrigorev) [inline week_5_batch_processing/setup/macos.md:30]: I'm not very familiar with bash on macOs, but shouldn't it be `.bashrc`? On Linux it's a more conventional way of updating the PATH variable

Comment (IremErturk) [reply week_5_batch_processing/setup/macos.md:30]: I am also using zsh, but by google search I have read that ~/.bashrc is used more for a non-interactive shells and ~/.bash_profile is used for user-based operations. therefore I am going to leave as it is

**Pull request merged**